docs/how-to/rocm-for-hpc/index.rst

```diff
@@ -0,0 +1,239 @@
+.. meta::
+   :description: How to use ROCm for HPC
+   :keywords: ROCm, AI, high performance computing, HPC, usage, tutorial
+
+******************
+Using ROCm for HPC
+******************
+
+The ROCm open-source software stack is optimized to extract the best high
+performance computing (HPC) workload performance from AMD Instinct™ accelerators
+while maintaining compatibility with industry software frameworks.
+
+ROCm enhances support and access for developers by providing streamlined and
+improved tools that significantly increase productivity. Being open-source, ROCm
+fosters innovation, differentiation, and collaboration within the developer
+community, making it a powerful and accessible solution for leveraging the full
+potential of AMD accelerators' capabilities in diverse computational
+applications.
+
+* For more information, see :doc:`What is ROCm? <../../what-is-rocm>`.
+
+* For guidance on installing ROCm, see :doc:`rocm-install-on-linux:index`. See
+  the :doc:`../../compatibility/compatibility-matrix` for details on hardware
+  and operating system support.
+
+Some of the most popular HPC frameworks are part of the ROCm platform, including
+those to help parallelize operations across multiple accelerators and servers,
+handle memory hierarchies, and solve linear systems.
+
+.. image:: ../../data/how-to/rocm-for-hpc/hpc-stack-2024_6_20.png
```

Comment:
Not sure about this - perhaps it was discussed but these boxes don't line up with the content below. 
![image](https://github.com/ROCm/ROCm/assets/164940351/c83f07f5-6131-4104-b057-9ac6ec2237b0)

Reply:
I'm not even sure I see the value of the image for this topic. 
This image might be of more use elsewhere in the documentation.